internal/rpc/context.go

```diff
@@ -131,6 +132,20 @@ func (m *metadataInterceptor) WrapStreamingHandler(req connect.StreamingHandlerF
 	return func(ctx context.Context, s connect.StreamingHandlerConn) error {
 		logger := log.FromContext(ctx)
 		logger.Tracef("%s (streaming handler)", s.Spec().Procedure)
+		// Intercept and log any panics, then re-panic.
+		defer func() {
```

Comment:
"metadataInterceptor" is intended just to propagate metadata headers. For panic recovery we should add another interceptor.